test/native/matchers/to-be-visible.js

```diff
@@ -17,7 +17,7 @@ function isStyleVisible( element ) {
 }
 
 function isAttributeVisible( element ) {
-	return element.type !== 'Modal' || ! element.props.visible === false;
+	return element.type !== 'Modal' || element.props.visible !== false;
```

Comment:
> Should this just be a truthy check, e.g., return element.type !== 'Modal' || !!element.props.visible;?

I thought about this option at the beginning, but I realized that it's possible that the prop is not defined, which in that case the default value (`true` as per the [RN documentation](https://reactnative.dev/docs/modal#visible)) is used. For this reason, I decided to focus on checking if it's not explicitly `false`.

> It's difficult to understand this prop out of context though. I was just thinking that it may be possible for visible to be undefined, and wasn't sure if that implied visibility. I also just noticed that all you did was shift the location of the logical not operator. So maybe don't worry about this comment since it's probably been in use without issue for a while now?

Yeah, the change was purely a fix in the condition. Actually, it wasn't related to the PR itself but I decided to apply it here as it's required for the tests.